### DISCUSSION TITLE
Python wheel dakota

### DISCUSSION THREAD

**Discussion by wvangeit** [snl-dakota/dakota#98] [Show and tell]
FYI, I have released a first draft for python wheel on PyPi that install the dakota environment module.
https://github.com/ITISFoundation/itis-dakota

```
pip install itis-dakota
````

Then you can execute a study with e.g.:
```
import dakota.environment as dakenv

dakota_conf = "your_dakota_configuration_file_content"
callbacks = {"model": your_evaluation_function}
study = dakenv.study(callbacks=callbacks, input_string=dakota_conf) 
study.execute()
```
(see https://github.com/snl-dakota/dakota/blob/devel/src/unit/test_dakota_python_env.py for an example how to use the environment module)

Wrt to the license of the wheel-building code, for now I've just put it as proprietary, but that's just because we still need to decide about which license to use. (and is open for discussion in general)

**Comment by jadamstephens** [comment]
This is great news! I will try to review it soon and offer feedback.
